src/components/OrganizationHeader/OrganizationHeader.tsx

```diff
@@ -76,23 +75,23 @@ const useStyles = makeStyles(theme => ({
     lineHeight: 1.2,
     fontWeight: 700,
     color: theme.colors.text,
-    margin: theme.spacing(6, 0),
+    margin: theme.spacing(5, 0),
   },
 }));
 
-interface Props {
-  buttonText: string;
-  onButtonClick: () => void;
-}
-export const OrganizationHeader = ({ buttonText, onButtonClick }: Props) => {
+export const OrganizationHeader = () => {
   const classes = useStyles();
   const navItems = getAdminNavigation();
   const currentOrg = useCurrentOrg();
 
   return (
     <>
-      <div className={classes.topMenu}>
-        <div className={classes.organizationLinks}>
+      <Box
+        css={{
+          display: 'flex',
+        }}
+      >
+        <Box css={{ flexGrow: 1 }} className={classes.organizationLinks}>
```

Comment:
We did agree that if one is making changes to components that we'd go stitches all the way. :D.  understand scope limitations thought

Reply:
thanks for the reminder; i can take this file all the way through. was just cutting corners because this was a diversion from what i was trying to get done at the time

Reply:
Yeah, totally understand. Don't mean to punish you but we have tech debt to pay :D